docs/user/metrics/features.csv

```diff
@@ -7,3 +7,4 @@ Library,ML Backend,I/O Backend,Spatial Backend,Transform Backend,Datasets,Weight
 `DeepForest`_,PyTorch,"GDAL, OpenCV, pandas, pillow, scipy",R-tree,Albumentations,0,3,❌,❌,❌,❌
 `SITS`_,R Torch,GDAL,-,tidyverse,22,0,❌,✅,✅,✅
 `TerraTorch`_,PyTorch,"GDAL, h5py, pandas, xarray",R-tree,Albumentations,16,1,✅,✅,❌,🚧
+`scikit-eo`_,"scikit-learn, TensorFlow*","pandas, scipy, numpy","geopandas, rasterio",numpy,22,0,❌,❌,❌,🚧
```

Comment:
Tell me more about the TensorFlow integration. Do all versions support TF? How tightly is TF integrated into the library design (like is it a core component or just used in some demo notebooks)?

Usually rasterio is used for I/O. By "spatial backend" we just mean the library used to track the bounding boxes of all data for sampling purposes, although I'm fine with a wider interpretation.

Also, where can I find the list of supported datasets?

Reply:
@yotarazona should be able to answer here

Reply:
Hi @adamjstewart and @Nowosad 

Scikit-eo uses TensorFlow only in the ```deeplearning.py``` implementation (https://github.com/yotarazona/scikit-eo/blob/main/scikeo/deeplearning.py) and only on a notebook: https://github.com/yotarazona/scikit-eo/blob/main/examples/notebooks/11_Deep_Learning_Classification_FullyConnected.ipynb. 
The ```deeplearning.py``` implementation was developed since the first version, but since version v0.2.29, the documentation was changed according to the latest updates of TensorFlow (i.e., the way to load the tensorflow library and keras). 

Regarding the I/O, got it, so rasterio is used as a I/O.
Regarding the "Spatial Backend", we put geopandas and raster because there is an implementation that allows to extract multivariate values from the spatial intersection of a satellite image and a vector spatial data (point feature for now). In any case, it can be left as “no support” in case it does not comply with the “Backend spatial” stipulations.

Right now we do not have a list of datasets but in the example notebooks you will find download links to the datasets used in each demo example. In most examples we use two datasets, a satellite image and a vector data. Need a list of datasets? We can insert a list in the README.md presentation of the examples.

Reply:
* ML Backend: Got it. In that case, it seems like TF is still supported, so you could probably remove the asterisk if you want
* I/O Backend: In this case I think it makes sense to list rasterio as an I/O backend and geopandas as a spatial backend
* Datasets: If I understand correctly, scikit-eo does not provide any builtin data loaders for existing datasets, it simply provides the model training code (and demonstrates how to create your own data loaders in the documentation). In that case, I would put 0 for # of datasets.